addons/mail/static/src/web/discuss/sidebar.xml

```diff
@@ -104,6 +104,9 @@
                 t-on-click.stop="() => this.leaveChannel(thread)" title="Leave this channel" role="img"/>
             <div t-if="threadService.canUnpin(thread)" t-attf-class="fa fa-times ms-1" t-on-click.stop="() => threadService.unpin(thread)" title="Unpin Conversation" role="img"/>
         </div>
+        <t t-if="Object.keys(thread.rtcSessions).length > 0">
```

Comment:
This t-if can go on the div itself now

Reply:
@phenix-factory updated, I think that it has been possible for a while but that it was a guideline in discuss to separate them